react/week3/todo-app/src/components/GetTodos.js

```diff
@@ -0,0 +1,61 @@
+import { useState, useEffect } from "react";
+import TodoList from "./TodoList";
+import AddTodo from "./AddTodo"
+
+const GetTodos = () => {
+    const API_URL = "https://gist.githubusercontent.com/benna100/391eee7a119b50bd2c5960ab51622532/raw";
+
+    const [todoData, setTododata] = useState([])
+    const [isLoading, setIsLoading] = useState(true);
+    useEffect(() => {
+        fetch(API_URL)
+            .then(res => res.json())
+            .then(todoArray => {
+                setIsLoading(prev => prev = false)
```

Comment:
You can set your state like: `setIsLoading(false)`. 
What your doing is useful, if you want to use the value from the previous state:
`setTimesLoaded(counter => counter + 1  )` <= This would increase the counter by 1.